### PR TITLE
Add some fields to Image

### PR DESCRIPTION
**Pull request by xuchenCN**
This PR adapt to api >=1.25

---
This change is [Reviewable](https://reviewable.io/reviews/docker-java/docker-java/887)

### CODE REVIEW

File: src/main/java/com/github/dockerjava/api/model/Image.java [hunk 1]
```diff
@@ -31,12 +32,24 @@
     @JsonProperty("RepoTags")
     private String[] repoTags;
 
+    @JsonProperty("RepoDigests")
+    private String[] repoDigests;
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/model/Image.java]: No arrays https://github.com/docker-java/docker-java/blob/master/docs/devel.adoc

Comment (xuchenCN) [reply src/main/java/com/github/dockerjava/api/model/Image.java]: This field just like "RepoTags" , so it defined to arrays , if it's incorrect please let me know the detail thanks

Comment (KostyaSha) [reply src/main/java/com/github/dockerjava/api/model/Image.java]: we are moving from arrays to List

Comment (KostyaSha) [reply src/main/java/com/github/dockerjava/api/model/Image.java]: Could you change it?

Comment (xuchenCN) [reply src/main/java/com/github/dockerjava/api/model/Image.java]: @KostyaSha Please check it

**Comment by bsideup** [comment]
Superseeded by #1260

**Pull request closed without merging**